File: Week 1/mario/mario.c
#include <stdio.h>
#include "cs50.c"

int main(void)
{
    // Get height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Calculation
    for (int n = 0; n < height; n++)
    {
        // Loop for number of space
        for (int x = 1; x < (height - n); x++)
        {
            printf(" ");
        }
        // Loop for number of #
        for (int y = -1; y < n; y++)
        {
            printf("#");
        }
        // New line character
        printf("\n");
    }
}

File: Week 6/Mario Less/mario.py
height = 0

# While loop for checking height
while (not height >= 1 or not height <= 8):
    try:
        height = int(input("Height: "))
    except ValueError:

        # Do Nothing
        pass

for i in range(1, height+1):

    # Printing blank space without new line character
    print(" "*(height-i), end="")

    # For loop for printing #
    for l in range(i):

        # Printing #
        print("#", end="")

    # Adding new line character
    print("\n", end="")